### PR TITLE
CUST-200 -Addling Update method

### PR DESCRIPTION
**Pull request by montescoro**
Adding Update method by Jorge Montes

### CODE REVIEW

**Review by montescoro — COMMENTED** [review]
I didn't put implementation in method

**Review by xernai — CHANGES_REQUESTED** [review]
Please add implementation

File: Etzmon.Customers.Web/Controllers/CustomersController.cs [hunk 1]
```diff
@@ -43,5 +43,11 @@ public IActionResult Insert([FromBody] Customer customer)
             //int trainingID = parameters.Get<int>("@CustomerID");    
             return View(customer);
         }
+
+        [HttpPut]
+        public IActionResult Update([FromBody] Customer customer)
+        {
```
Comment (xernai) [inline Etzmon.Customers.Web/Controllers/CustomersController.cs:49]: Add implementation

Comment (montescoro) [reply Etzmon.Customers.Web/Controllers/CustomersController.cs:49]: ok

**Pull request merged**